src/ResourceManager/Sql/Commands.Sql/help/Stop-AzureRmSqlDatabaseActivity.md

```diff
@@ -0,0 +1,165 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+Module Name: AzureRM.Sql
+ms.assetid: B5C909D7-6087-463A-83BF-99DD196B9862
+online version: 
+schema: 2.0.0
+---
+
+# Stop-AzureRmSqlDatabaseActivity
+
+## SYNOPSIS
+Cancels the asynchronous operation on the database.
```

Comment:
@jiawang003 what kind of operations are being canceled by this cmdlet?

Reply:
added word "updateslo"